packages/atlas/src/components/_comments/Comment/Comment.styles.ts

```diff
@@ -85,3 +86,23 @@ export const CommentFooterItems = styled.div`
 export const StyledFooterSkeletonLoader = styled(SkeletonLoader)`
   border-radius: 999px;
 `
+
+export const RepliesWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`
+
+export const StyledAvatarGroup = styled(AvatarGroup)`
+  margin-left: ${sizes(3)};
+`
+
+export const ReplyButton = styled(Button)`
+  position: relative;
+  top: 1px;
```

Comment:
Hmm. I'm curious. Why do we need to additional 1px top? I also noticed that you also applied `align-items: center` on the wrapper, but it looks like it does nothing, can you double-check why this is not working? 

Just to clarify - If there is no easy solution to get rid of this I'm fine with leaving it as it is now.